src/python/Utils/IteratorTools.py

```diff
@@ -30,24 +31,24 @@ def nestedDictUpdate(d, u):
     Code from Alex Matelli
     http://stackoverflow.com/questions/3232943/update-value-of-a-nested-dictionary-of-varying-depth
     """
-    for k, v in u.iteritems():
+    for k, v in viewitems(u):
         if isinstance(v, collections.Mapping):
             r = nestedDictUpdate(d.get(k, {}), v)
             d[k] = r
         else:
             d[k] = u[k]
     return d
 
-def convertFromUnicodeToStr(data):
+def convertFromUnicodeToBytes(data):
```

Comment:
Dario, I believe you did, but let me confirm it. Did you grep the whole code for other places calling/importing this function?

Reply:
Yep, I did grep all the code and did not find usage of this function.

```
> grep -r "convertFromUnicodeToStr" .
Binary file ./src/python/Utils/IteratorTools.pyc matches
./src/python/Utils/IteratorTools.py:def convertFromUnicodeToStr(data):
./src/python/Utils/IteratorTools.py:        return dict(list(map(convertFromUnicodeToStr, data.iteritems())))
./src/python/Utils/IteratorTools.py:        return type(data)(list(map(convertFromUnicodeToStr, data)))
>
```